open_mastr/mastr.py

```diff
@@ -295,8 +300,8 @@ def to_csv(
         limit: None or int
             Limits the number of exported data and location units.
         """
+        log.info("Starting csv-export")
 
-        create_data_dir()
```

Comment:
Can we delete this function here @chrwm ?

Reply:
Yes I think so.